### ISSUE TITLE
Feature Request: Add mp3 metadata before downloading

### ISSUE THREAD

**Issue by mhaffner** [mps-youtube/mps-youtube#754]
I really dig this tool, especially the download feature. My one complaint is that after I download .mp3's, there is no metadata attached, and thus, my ncmpcpp library it does not recognize the artist or album name. Of course, I wouldn't really expect it to unless this data was built into Youtube and easily accessible to mpsyt. Would there be a way to designate an album/artist name to tracks and then apply it to downloads?

Related - I've read about numerous mp3 metadata editing tools, but these all incur too much manual labor for me. Is there a way to post hoc edit multiple mp3's metadata as text? This would be a suitable workaround for me. (I do something similar to rename large batches of files in Emacs with Dired occasionally)

**Comment by vn-ki** [comment]
You're talking something along the lines of #740, right? I'll make a new PR when I'm free.

**Comment by mhaffner** [comment]
@vn-ki Yup. That's exactly what I'm talking about. Thanks for the quick reply.
